ERROR    logger:requests_util.py:34 【退出登录】接口请求异常，响应体：{'status': 500, 'msg': '用户未登录', 'data': []}